File: LP/Exercícios/ficha1.c
#include <stdio.h>

int mdc(a, b)
{
    if (a == b)
        return a;
    else if (b > a)
        return mdc(a, b - a);
    else 
        return mdc(a - b, b);     
}

int main()
{
    int a, b;

    scanf("%d %d", &a, &b);
    mdc(a, b);
    return 0;
    
}

File: LP/Exercícios/ficha1.java
package LP.aulas;

import java.io.IOException;

public class MDC {

    public static int mdc(int a , int b){
        if (a == b)
            return a;
        else if( b > a)
            return mdc(a, b - a);
        else 
            return mdc(a -b, a);
    }
    public static void main(String[] args) throws IOException {
        int a = Integer.parseInt(args[0]);
        int b = Integer.parseInt(args[1]);
        System.out.println(mdc(a, b));
    }
}